roles/strongswan/tasks/openssl.yml

```diff
@@ -155,10 +155,27 @@
         format: OpenSSH
       with_items: "{{ users }}"
 
+    - name: Gather the package facts
+      ansible.builtin.package_facts:
+        manager: auto
+
+    - name: Get OpenSSL version
+      shell: |
+        set -o pipefail
```

Comment:
When running `cloud-init` (Ubuntu 22.04 LTS x64 on Vultr), I got an error because `set -o pipefail` fails on `/bin/sh`

Fixed by specifying which shell to run:
```yml
  # (...)
  shell: |
        set -o pipefail
        {{ openssl_bin }} version |
        cut -f 2 -d ' '
  args:
    executable: bash
  # (...)
```

Reply:
@rafaelsms, it looks like the comment was fixed. Do you think we should merge it now? Certs generated with the current state of the master branch couldn't be imported into the latest macOS / iOS version. Apple still needs to update its libs, and certs generated via openssl@3 are incompatible with the latest Apple OS.